tf_files/aws/modules/commons-alarms/variables.tf

```diff
@@ -0,0 +1,30 @@
+# id of AWS account that owns the public AMI's
+
+variable "slack_webhook" {
+  default = "https://hooks.slack.com/services/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"
```

Comment:
Don't post this in public channels, make the default value an empty string `""` and then in `config.tfvars` we can set it up.

You could tweak the `workon` script to automatically fill the value up in `config.tfvars` automatically, but that's completely optional.